kuberenetes/standalone.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: mongoose
+  namespace: mongoose
+spec:
+  containers:
+    - image: emcmongoose/mongoose
+      name: mongoose
+      command: ["/bin/sh"]
```

Comment:
this may be ommitted and the default command will be used running the entrypoint

Reply:
Yes, it is really optional. I saw such a record from the guys from the flex.